app/view/show.py

```diff
@@ -13,6 +13,32 @@
 from collections import defaultdict
 
 
+def add_bug_link(cves, pkgs, severity):
```

Comment:
i think, since we return the pure data now instead of a link, this should be named like "get_bug_data()"?